.github/workflows/gh-pages.yml

```diff
@@ -32,10 +32,16 @@ jobs:
           python -m pip install --upgrade pip
           python -m pip install -r requirements.txt -r requirements-dev.txt
           python -m pip install '.[all]'
+      - uses: actions/cache@v4
+        id: cache-docs
+        with:
+          path: docs/_examples_database/
+          key: ${{ runner.os }}-docs_cache
```

Comment:
Why include the OS in the cache key? Is there a specific reason for that?

Reply:
Ok. I *think* the cache can be unique across OSes. That would allow us to always generate the cache on Linux (the cheaper runner) and re-use it on Windows and Mac (that are up to 10x more expensive).